src/main/java/com/google/sps/data/CommentTable.java

```diff
@@ -28,20 +29,41 @@ public void addEntry(String name, String comment) {
             return;
         }
 
+        if (checkInjection(name, comment)) {
+            return;
+        }
+
         Entity commentEntity = new Entity(ENTRY_KEY);
         commentEntity.setProperty(NAME_KEY, name);
         commentEntity.setProperty(COMMENT_KEY, comment);
 
         datastore.put(commentEntity);
     }
 
+    private boolean checkInjection(String name, String comment) {
```

Comment:
nit: As a convention within Google, methods that start with the verb "check" throw an error if the check fails. It's like an assertion. I would suggest either:
1. Make it throw an exception and catch the exception in your doPost call.
2. Rename to something like isSafe() and return a boolean